### PR TITLE
accessibility - edit heading levels and id labels

### PR DESCRIPTION
**Pull request by emma-r-slight**
## Description

Changed <h_> heading to descending order(and changed CSS to match where needed).  Also changed Id labels to use "item".id instead of the map "index" number so each Aria-id is unique across all pages.

## Related Issue

[Issue 221](https://github.com/Vets-Who-Code/vets-who-code-app/issues/221)
[Issue 220](https://github.com/Vets-Who-Code/vets-who-code-app/issues/220)
(https://github.com/Vets-Who-Code/vets-who-code-app/issues/223)

## How Has This Been Tested?

Used existing tests new tests not needed.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by emma-r-slight** [comment]
I have been through and tested each page with lighthouse and they all pass with 100 now.

File: src/components/Forms/ApplyForm/ApplyForm.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ function ApplyForm() {
       </div>
       <div className="col-md-8">
         <div className="form-group">
-          <label htmlFor="lastNname" className="dark-text">
+          <label htmlFor="lastName" className="dark-text">
```
Comment (gixxerblade) [inline src/components/Forms/ApplyForm/ApplyForm.js:99]: Please add an `aria-label="last name"`

File: src/components/Forms/ApplyForm/ApplyForm.js [hunk 2]
```diff
@@ -113,7 +113,7 @@ function ApplyForm() {
       </div>
       <div className="col-md-8">
         <div className="form-group">
-          <label htmlFor="InputEmail" className="dark-text">
+          <label htmlFor="email" className="dark-text">
```
Comment (gixxerblade) [inline src/components/Forms/ApplyForm/ApplyForm.js:116]: Please add an `aria-label="email"`

**Review by gixxerblade — CHANGES_REQUESTED** [review]
Thank you for your PR. We really appreciate anyone's help with maintaining our site. I added a couple items for your review. Also please update the snapshot with `yarn test --update-snapshot`

File: src/components/Forms/ApplyForm/ApplyForm.js (hunk 1 above)
Comment (emma-r-slight) [reply src/components/Forms/ApplyForm/ApplyForm.js:99]: I can add aria-label to these labels, do you want me to add it to all the labels on the page?  From my understanding of Aria specifications, adding aria-label is not needed here. You have used the correct semantic HTML with the appropriate attributes therefore by default these elements are accessible.  

I recently learned that although we use a variable type name to identify the input and label, when labeled correctly what is actually read by a screen reader is what is between the label tags.  <label .....attributes>Read this</label>.  Adding aria-label and aria-labelledby only needs to be added if for whatever reason there is no label element for you input element.

Comment (emma-r-slight) [reply src/components/Forms/ApplyForm/ApplyForm.js:99]: This is a fantastic  resource for accessibility best practice - [Accessibility Oz](https://www.accessibilityoz.com/videos/)
I have it bookmarked and refer back to it all the time!

**Review by gixxerblade — COMMENTED** [review]
Emma, can you please rebase with the upstream to resolve conflicts, please? Again thank you for contributing!

**Review by gixxerblade — APPROVED** [review]
Thank you!

**Pull request merged**